DIAGRAMS.md

```diff
@@ -0,0 +1,304 @@
+# Diagrams
+
+# Project Structure
+Kyoo is a monorepo that consists of several projects each in their own directory.  Diagram below shows an outline of kyoo, projects, and artifacts.
+
+```mermaid
+block-beta
+  columns 1
+  block:proj1:1
+    proj_name["Kyoo"]:1
+  end
+  block:proj2:1
+    dir_1["autosync/"]
+    dir_2["back/"]
+    dir_3["front/"]
+    dir_4["transcoder/"]
+    dir_5["scanner/"]
+  end
+  block:proj3:1
+    %% columns auto (default)
+    block:autosync_b:1
+      autosync_i1("kyoo_autosync")
+    end
+    block:back_b:1
+      columns 1
+      back_i1("kyoo_back")
+      back_i2("kyoo_migrations")
+    end
+    block:front_b:1
+      front_i1("kyoo_front")
+    end
+    block:transcoder_b:1
+      transcoder_i1("kyoo_transcoder")
```

Comment:
why is this an half block?

Reply:
Definitely have a desire for each component to be the same size.  I do not see a way to do it within the block diagram atm.